cli.js

```diff
@@ -21,15 +21,32 @@ const main = async ({
   token = null,
   delegates = null,
   port = 3000,
-  loglevel = "error",
+  loglevel = "info",
```

Comment:
Is this a good default though ? It will add a lot of noise in js-ipfs tests unless opted in there.

Reply:
`info` is only used in `cli.js` :) 
IIUC tests in js-ipfs use programmatic interface and that one remains on `error` level by default:

https://github.com/ipfs-shipyard/js-mock-ipfs-pinning-service/blob/f3f99082291c9ebf1612ec0f3848e5482ccb7314/index.js#L29

So there  should not be any regression there.